pkg/cri/resource-manager/resource-manager.go

```diff
@@ -78,8 +78,13 @@ func NewResourceManager() (ResourceManager, error) {
 		return nil, err
 	}
 
-	if opt.ResetPolicy {
+	switch {
+	case opt.ResetPolicy && opt.ResetConfig:
+		os.Exit(m.resetCachedPolicy() + m.resetCachedConfig())
+	case opt.ResetPolicy:
```

Comment:
in theory, here 'fallthrough' can be used instead of trying to do sum of `error` type.